build.rs

```diff
@@ -1,8 +1,29 @@
 use std::{env, path::Path};
 
 fn main() {
-    println!(
-        "cargo:rustc-env=HANDLER={}",
-        AsRef::<Path>::as_ref(&env::var_os("DEP_SENTRY_NATIVE_HANDLER").unwrap()).display()
-    );
+    if let Ok(handler) = env::var("DEP_SENTRY_NATIVE_HANDLER") {
+        if cfg!(feature = "copy-handler") {
+            let out_dir = env::var("OUT_DIR").expect("out dir not set");
+
+            let out_dir = Path::new(&out_dir);
+            let bin_dir = out_dir
```

Comment:
this `../../../` chain is a bit confusing. can you maybe give an example in a comment what happens here?

Reply:
Added an explanation for why it goes up 3 directories.